_team_and_board/5-noffle.md

```diff
@@ -0,0 +1,27 @@
+---
+categories:
+  - team
+layout: team
+title: Stephen Whitmore
+role: "Developer"
+twitter: "https://twitter.com/noffle"
+tags:
+  - core
+current: true
+permalink: /team/noffle
+image: "/assets/noffle.png"
+---
+
+**Stephen Whitmore** works with Digital Democracy to build useful
+open tools that raise the bar of the commons, especially for marginalized
+communities.
+
+Stephen creates and maintains open technology that enables self-determination;
+honours people, not profit; is sustainable for the communities that adopt it;
+and is maximally accessible regardless of resources or technical background.
+
+Contact: [sww@eight45.net](mailto:sww@eight45.net)
```

Comment:
It redirects to the above address. *However*, providers like GMail don't trust my non-blessed mail domain, so they often filter my emails if the receiver hasn't heard of me before. If they use the above address the first time around, my address will be blessed and they'll get my future correspondence. What a terrible state email is in. :cry: